2.train_model/README.md

```diff
@@ -2,7 +2,10 @@
 
 In this module, we train ML models to predict phenotypic class from cell features.
 
-We train the models in [train_model.ipynb](1.train_model.ipynb).
+In [train_multi_class_models.ipynb](train_multi_class_models.ipynb), we train models to predict which phenotypic class the cell features are from out of 15 possible classes (anaphase, metaphase, apoptosis, etc).
+In [train_single_class_models.ipynb](train_single_class_models.ipynb), we train models to predict whether or not the cell features are from a particular phenotypic class or not.
```

Comment:
```suggestion
In [train_single_class_models.ipynb](train_single_class_models.ipynb), we are  training models to predict whether the cell features are from a particular phenotypic class or not.
```

Reply:
I kept my verb tense but included your other suggestion in https://github.com/WayScience/phenotypic_profiling_model/pull/25/commits/6038b55195d1339937b3f667eb9ed2400677c875